src/itest/resources/features/company-metrics-appointments.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Appointments acceptance criteria
+
+  Scenario Outline: Create
+    Given A company metrics resource does not exist for a company number of "0123456"
+    And The company has ACTIVE appointments for <activeDirectors> directors, <activeSecretaries> secretaries, <activeLLPMembers> LLP members, <activeCorporateLLPMembers> Corp. LLP members
+    And The company has RESIGNED appointments for <resignedDirectors> directors, <resignedSecretaries> secretaries, <resignedLLPMembers> LLP members, <resignedCorporateLLPMembers> Corp. LLP members
+    And The user is authenticated and authorised with internal app privileges
+    When The recalculate endpoint is called by user "userId"
+    Then The response code should be HTTP OK
+    And The location response header should be "/company/0123456/metrics"
+    And The response body should be empty
+    And The metrics resource has a count.appointments object
+    And The appointments should have <activeDirectors> active directors, <activeSecretaries> active secretaries, <activeLLPMembers> active LLP members, <activeCorporateLLPMembers> active Corp. LLP members, <totalCount> total appointments, <activeCount> active appointments, <resignedCount> resigned appointments
+    And The etag should be updated
+    And The updated object will have valid timestamp (UTC), type : "company_metrics" and by "userId"
```

Comment:
is the "userId" here the same as "context_id"? Might be confusing for someone down the line to see this being passed in in the updated_by if the context_id doesn't actually represent a user.

Reply:
See latest commit